eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Pool.hs

```diff
@@ -172,15 +173,24 @@ instance Era era => DecCBOR (ShelleyPoolPredFailure era) where
       k -> invalidKey k
 
 poolDelegationTransition ::
-  forall era.
-  (ShelleyEraTxCert era, EraPParams era) =>
-  TransitionRule (ShelleyPOOL era)
+  forall (ledger :: Type -> Type) era.
+  ( ShelleyEraTxCert era
+  , EraPParams era
+  , Signal (ledger era) ~ TxCert era
```

Comment:
I think it'd be a good idea to break this transition function into two smaller functions, one that takes a TxCert that we can use in Shelley, and then another one that only looks at the PoolCert cases that we can use in Conway. That way we can use PoolCert as the signal in Conway.

Reply:
Done!